### PR TITLE
added systemd collector config

### PR DESCRIPTION
**Pull request by ehienabs**
Docs for the Systemd collector 
https://github.com/fluent/fluent-bit/pull/7319

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204737714980300

### CODE REVIEW

File: pipeline/inputs/node-exporter-metrics.md [hunk 1]
```diff
@@ -37,6 +37,11 @@ This plugin is currently only supported on Linux based operating systems\
...
 | filesystem.ignore\_filesystem\_type\_regex  | Specify the regex for the filesystem types to prevent collection of/ignore. | `^(autofs|binfmt_misc|bpf|cgroup2?|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|iso9660|mqueue|nsfs|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|selinuxfs|squashfs|sysfs|tracefs)$`    |
 | diskstats.ignore\_device\_regex | Specify the regex for the diskstats to prevent collection of/ignore. | `^(ram|loop|fd|(h|s|v|xv)d[a-z]|nvme\\d+n\\d+p)\\d+$` |
+| systemd_service_restart_metrics | Determines if the collector will include service restart metrics | false |
+| systemd_unit_start_time_metrics | Determines if the collector will include unit start time metrics | false |
+| systemd_include_service_task_metrics | Determines if the collector will include service task metrics | false |
+| systemd_include_pattern | regex to determine which units are included in the metrics produced by the systemd collector | it's not applied unless explicitly set |
```
Comment (patrick-stephens) [inline pipeline/inputs/node-exporter-metrics.md:43]: I'm (apologies for the pun) not a huge fan of contractions like `it's` particularly when this is further confused by possessive forms. Let us try to ensure we use the full set of words, it is easy to accidentally misread or typo with a contraction.
```suggestion
| systemd_include_pattern | regex to determine which units are included in the metrics produced by the systemd collector | It is not applied unless explicitly set |
```

File: pipeline/inputs/node-exporter-metrics.md [hunk 2]
```diff
@@ -37,6 +37,11 @@ This plugin is currently only supported on Linux based operating systems\
...
+| systemd_exclude_pattern | regex you'd set to determine which units are excluded in the metrics produced by the systemd collector | `.+\\.(automount|device|mount|scope|slice)"`    |
```
Comment (patrick-stephens) [inline pipeline/inputs/node-exporter-metrics.md:44]: As above, try to remove contractions.
```suggestion
| systemd_exclude_pattern | regex to determine which units are excluded in the metrics produced by the systemd collector | `.+\\.(automount|device|mount|scope|slice)"`    |
```

File: pipeline/inputs/node-exporter-metrics.md [hunk 3]
```diff
@@ -65,6 +71,7 @@ The following table describes the available collectors as part of this plugin. A
 | time      | Exposes the current system time.                                                                 | Linux  | v1.8    |
 | uname     | Exposes system information as provided by the uname system call.                                 | Linux  | v1.8    |
 | vmstat    | Exposes statistics from `/proc/vmstat`.                                                          | Linux  | v1.8.2  |
+| systemd collector | Exposes statistics fr systemd.                                                           | Linux  | v1.8.2  |
```
Comment (patrick-stephens) [inline pipeline/inputs/node-exporter-metrics.md:74]: I think this is more recent than 1.8.2? 2.1.4?

Comment (cosmo0920) [reply pipeline/inputs/node-exporter-metrics.md:74]: Systemd collector was introduced in 2.1.3:
https://github.com/fluent/fluent-bit/commit/ab60c002ae9edae6548ee129f9f38cecf7732239

**Review by patrick-stephens — APPROVED** [review]

File: pipeline/inputs/node-exporter-metrics.md [hunk 4]
```diff
@@ -65,6 +71,7 @@ The following table describes the available collectors as part of this plugin. A
 | time      | Exposes the current system time.                                                                 | Linux  | v1.8    |
 | uname     | Exposes system information as provided by the uname system call.                                 | Linux  | v1.8    |
 | vmstat    | Exposes statistics from `/proc/vmstat`.                                                          | Linux  | v1.8.2  |
+| systemd collector | Exposes statistics fr systemd.                                                           | Linux  | v2.1.3  |
```
Comment (patrick-stephens) [inline pipeline/inputs/node-exporter-metrics.md:74]:
```suggestion
| systemd collector | Exposes statistics from systemd.                                                           | Linux  | v2.1.3  |
```

**Pull request merged**